src/Components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { FilterOption } from '../../enums/FilterOption';
+
+interface Props {
+  filterOption: FilterOption;
+  setFilterOption: (filterOption: FilterOption) => void;
+}
+
+export const TodoFilter: FC<Props> = ({
+  filterOption,
+  setFilterOption,
+}) => {
+  const filterOptions = Object.keys(FilterOption);
```

Comment:
Let's use entries to avoid `const filterOptionKey = option as keyof typeof FilterOption;`

```suggestion
  const filterOptions = Object.Entries(FilterOption);
```